src/test/java/org/broadinstitute/hellbender/utils/recalibration/QualQuantizerUnitTest.java

```diff
@@ -16,7 +16,7 @@
 
 
 public final class QualQuantizerUnitTest extends BaseTest {
-    @BeforeSuite
+    @BeforeClass
     public void before() {
```

Comment:
is this an empty method?

Reply:
Hm, yeah, that should probably just come out. Even better.

Reply:
Done.